modules/olm-cs-status-cli.adoc

```diff
@@ -0,0 +1,119 @@
+// Module included in the following assemblies:
+//
+// * operators/admin/olm-status.adoc
+// * support/troubleshooting/troubleshooting-operator-issues.adoc
+
+ifdef::openshift-origin[]
+:global_ns: olm
+endif::[]
+ifndef::openshift-origin[]
+:global_ns: openshift-marketplace
+endif::[]
+
+[id="olm-cs-status-cli_{context}"]
+= Viewing Operator catalog source status using the CLI
+
+You can view the status of an Operator catalog source using the CLI.
+
+.Prerequisites
+
+* You have access to the cluster as a user with the `cluster-admin` role.
+* You have installed the OpenShift CLI (`oc`).
+
+.Procedure
+
+. List the catalog sources for the namespace where your catalog source was created. For example, you can check the `{global_ns}` namespace, which is used for cluster-wide catalog sources:
```

Comment:
The repetition of "catalog source" in the first sentence seems a little awk.  Obligatory sugg:

. List the catalog sources in a namespace. For example,...

If you prefer the orig, maybe "...where you created the catalog source."